vmdb/app/models/miq_event.rb

```diff
@@ -192,8 +192,8 @@ def self.event_name_for_target(target, event_suffix)
   end
 
   def miq_policies
-    p_ids = MiqPolicyContent.find_all_by_miq_event_id(self.id).collect {|pc| pc.miq_policy_id}.uniq
-    MiqPolicy.find(:all, :conditions => {:id => p_ids})
+    p_ids = MiqPolicyContent.where(:miq_event_id => self.id).uniq.pluck(:miq_policy_id)
+    MiqPolicy.where(:id => p_ids).to_a
```

Comment:
I think MiqPolicy objects are gettable from their content, so the two methods could be made into 1...(maybe separate PR)

```ruby
MiqPolicyContent.where(:miq_event_id => self.id).includes(:miq_policy).collect(&:miq_policy).uniq
```

ASIDE: Can you pluck relationships or is it strictly columns?